lib/widgets/twake_components/twake_preview_link/twake_link_preview_item.dart

```diff
@@ -60,12 +63,14 @@ class TwakeLinkPreviewItem extends StatelessWidget {
       return LinkPreviewLarge(
```

Comment:
WDYT : 
- You don't have to pass previewLink to children

```dart
    return InkWell(
      onTap: () {
        if (previewLink == null) return;
        UrlLauncher(context, url: previewLink).launchUrl();
      },
      child: urlPreviewPresentation.imageHeight! > 200
          ? LinkPreviewLarge(
              key: linkPreviewLargeKey, urlPreviewPresentation: urlPreviewPresentation)
          : LinkPreviewSmall(
              key: linkPreviewSmallKey,
              urlPreviewPresentation: urlPreviewPresentation,
            ),
    );
```

Reply:
you're right

Reply:
done